src/Pages/DetailPage/Components/ReservationBox/ReservationBox.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { ButtonBox } from './Components/ButtonBox';
+import { ReservationButton } from './Components/ReservationButton';
+import { PriceDetail } from './Components/PriceDetail';
+
+export const ReservationBox = ({ data }) => {
+  const [startDate, setStartDate] = useState(null);
+  const [endDate, setEndDate] = useState(null);
+  const [focusedInput, setFocusedInput] = useState(null);
+  const [openGuestBox, setOpenGuestBox] = useState(false);
+  const [guest, setGuest] = useState({
+    adult: 1,
+    child: 0,
+    kid: 0,
+  });
+  const [dateDiff, setDateDiff] = useState(0);
+  // const [hideGuestBox, setHideGuestBox] = useState(false);
+  // const guestBoxRef = useRef(null);
+
+  // useEffect(() => {
+  //   document.addEventListener('click', toggleShowGuestBox, false);
+  //   return () => {
+  //     document.removeEventListener('click', toggleShowGuestBox, false);
+  //   };
+  // }, []);
+
+  // const toggleShowGuestBox = evt => {
+  //   if (guestBoxRef.current && !guestBoxRef.current.contains(evt.target)) {
+  //     setHideGuestBox(true);
+  //   }
+  // };
+
+  const onDatesChange = ({ startDate, endDate }) => {
+    setStartDate(startDate);
+    setEndDate(endDate);
+    if (startDate && endDate) {
+      const diff =
+        Math.ceil(Math.abs(endDate._d.getTime() - startDate._d.getTime())) /
+        (1000 * 3600 * 24);
+      setDateDiff(diff);
+    }
+  };
+
+  const onFocusChange = focusedInput => {
+    setFocusedInput(focusedInput);
+  };
+
+  const toggleGuestBox = () => {
+    setOpenGuestBox(prev => !prev);
+  };
+
+  const controllQuantity = ({ target }, selectedOne) => {
+    const { alt } = target;
```

Comment:
```suggestion
  const controllQuantity = (e, selectedOne) => {
    const { alt } = e.target;
```
제 생각에는 이벤트 객체라는 점을 드러내는게 더 좋아보이는데 어떻게 생각하시나요?